docs/README.md

```diff
@@ -35,6 +35,7 @@
 * `/jobs` - [Jobs](jobs.md)
   * [Konnectors](konnectors.md)
   * [Workers](workers.md)
+- `/move` - [Move, export and import an instance](move.md)
```

Comment:
Should it be part of the PR? :)

Reply:
No, it's not related. It was just that I was working on fixing a typo on the sharings docs, and I saw this entry was missing. I added too in a commit about documentation.